test/e2e/external_gateways.go

```diff
@@ -984,6 +1153,27 @@ func pokeHostnameViaNC(podName, namespace, protocol, target string, port int) st
 	return hostname
 }
 
+// pokeConntrackEntries returns the number of conntrack entries that match the provided pattern, protocol and podIP
+func pokeConntrackEntries(nodeName, podIP, protocol string, patterns []string) int {
+	args := []string{"get", "pods", "--selector=app=ovs-node", "--field-selector", fmt.Sprintf("spec.nodeName=%s", nodeName), "-o", "jsonpath={.items..metadata.name}"}
+	ovsPodName, err := framework.RunKubectl("ovn-kubernetes", args...)
+	framework.ExpectNoError(err, "failed to get the ovs pod on node %s", nodeName)
+	args = []string{"exec", ovsPodName, "--", "ovs-dpctl", "dump-conntrack"}
+	conntrackEntries, err := framework.RunKubectl("ovn-kubernetes", args...)
+	framework.ExpectNoError(err, "failed to get the conntrack entries from node %s", nodeName)
+	numOfConnEntries := 0
+	for _, pattern := range patterns {
+		for _, connEntry := range strings.Split(conntrackEntries, "\n") {
+			if strings.Contains(connEntry, pattern) && strings.Contains(connEntry, protocol) && strings.Contains(connEntry, podIP) {
+				klog.Infof("SURYA %v", connEntry)
```

Comment:
@tssurya did you mean to leave this in there? :)

Reply:
oopsy :D good eye detail haha, let me remove it.